docs/Reference-for-template.json.md

```diff
@@ -83,12 +83,12 @@ There are 3 places from which a symbol can acquire its value:
 |---|---|--|
 |sourceName| The name in the source tree to replace with the name the user specifies. The value to be replaced with can be given using the `-n` `--name` options while running a template. The template engine will look for any occurrance of the name present in the config file and replace it in file names and file contents. If no name is specified by the host, the current directory is used. The value of the `sourceName` is available in built-in `name` symbol and can be used as the source for creating other symbols and condition expressions. || 
 |placeholderFilename|A filename that will be completely ignored expect to indicate that its containing directory should be copied. This allows creation of empty directory in the created template, by having a corresponding source directory containing just the placeholder file. Completely empty directories are ignored.|If not specified, a default value of `"-.-"` is used.|
-|primaryOutputs|A list of important output paths created during template generation. These paths need to be added to the newly created project at the end of template creation. It's defined as an array of [Primary Output](#primary-output-definition)||
+|primaryOutputs|A list of template files that are used in post actions. The path should contain the relative path to the file prior to the renaming that may happen during template generation. It's defined as an array of [Primary Output](#primary-output-definition)||
 
 #### Primary Output Definition
 |Name|Description|
 |---|---|
-|path|One instance of a primary output path.|
+|path|Contains the relative path to the file in the template definition.|
```

Comment:
'path to **the file** in'
Shouldn't this be '**a file**' ? Is there always one specific file?